modal_proto/api.proto

```diff
@@ -2373,6 +2373,17 @@ message SandboxListResponse {
   repeated SandboxInfo sandboxes = 1;
 }
 
+message SandboxGetResourceUsageRequest {
```

Comment:
If the proto is approved I'll split it out into its own PR so that the server-side can be merged before merging this PR.